src/webapp/webhook_status_messages.py

```diff
@@ -0,0 +1,86 @@
+
+dt_success = None
+
+dt_not_up_to_date = (
+    "Greetings @{sender},\n"
+    "\n"
+    "Thank you for your pull request!\n"
+    "\n"
+    "Your downtime update looks clean, but `{head_label}` ({head_sha})"
+    " was not up-to-date with `{base_label}` ({base_sha});"
+    " OSG Staff will need to manually approve this PR.\n"
+    "\n"
+    "-- OSG-BOT  :broken_heart:"
+)
+
+dt_not_all_checks_pass = (
+    "Greetings @{sender},\n"
+    "\n"
+    "Thank you for your pull request.\n"
+    "\n"
+    "Your [requested changes]({base_sha}...{head_sha}) affect one or more"
+    " downtime files, but are not eligible for downtime automerge;"
+    " OSG Staff will need to manually review this PR.\n"
+    "\n"
+    "-- OSG-BOT  :mag:\n"
+    "\n"
+    "---\n"
+    "\n"
+    "Output from the automerge downtime check script:\n"
+    "```\n"
+    "{stdout}\n"
+    "```\n"
+    "```\n"
+    "{stderr}\n"
+    "```\n"
+)
+
+non_dt_contact_error = (
+    "Greetings @{sender},\n"
+    "\n"
+    "Thank you for your pull request.\n"
+    "\n"
+    "Just a head's up - I did not find you in our contact database;"
```

Comment:
```suggestion
    "Just a heads up - I did not find you in our contact database;"
```

Reply:
When writing error messages, let's try and provide actionable items for users.

> We didn't find you in the contact db, here's how you get into the contact db.
> OSG staff will have to review this pull request manually.

Reply:
tbh, despite greeting the unknown user, that message was more to alert us osg staff than for the user.  Anyway, is there a link or something?  I have no idea what we recommend to the user in this case.

Reply:
https://opensciencegrid.org/docs/common/registration/#registering-contacts

Reply:
Thanks!  Updated.